Enable dynamic input switching

utzcoz opened the pull request:
The `vendor_maruos` part to enable dynamic input switching. For the whose view, please visit the issue [Enable dynamic input switching](https://github.com/maruos/maruos/issues/97).

utzcoz committed:
Enable input dynamic input switching

Remove the execluded input devices for Android and add methods to notify
the desktop to enable/disable input.

Signed-off-by: utzcoz <utzcoz@outlook.com>

utzcoz committed:
Add lxc-attach to Android to debug usage

Signed-off-by: utzcoz <utzcoz@outlook.com>

pdsouza commented on perspective/LXCContainerManager.cpp:
```diff
@@ -116,6 +118,24 @@ static bool containerIsRunning(const char* id) {
+
+    const char *program = enable ? "/etc/maruos/enable-input" : "/etc/maruos/disable-input";
+    lxc_attach_options_t attach_options = LXC_ATTACH_OPTIONS_DEFAULT;
+    // The lxc_attach_command_t.argv in attach_options.h needs including
```
Nice work figuring out how to use `lxc-attach`!

pdsouza commented on device-maru.mk:
```diff
@@ -38,6 +34,7 @@ PRODUCT_PACKAGES += \
 	liblxc \
 	lxc-start \
 	lxc-stop \
+	lxc-attach \
```
Yes, this is good for debugging - thanks.

pdsouza reviewed (approved):
Great work @utzcoz! Your PR to do dynamic input switching has touched basically all the core parts of Maru!

utzcoz commented:
@pdsouza  I have squashed all commits with two main commits for cleaning up.

utzcoz committed:
Use minimum namespaces when running lxc-attach

Mostyly we want to drop PID namespace to support devices
with old kernel that doesn't support PID namespace correctly.

Signed-off-by: utzcoz <utzcoz@outlook.com>

utzcoz commented:
@pdsouza I have added a new patch to drop `PID` namesapce when running lxc-attach.

pdsouza commented:
@utzcoz Merged - thanks for your patience!
